### PR TITLE
Kiera Thomasson - Calculator - Octos

### PR DESCRIPTION
**Pull request by Krashaune**
# Calculator
Congratulations! You're submitting your assignment.

## Comprehension Questions

|  Question 	|  Answer 	|
|---	|---	|
|  Describe how you stored user input in your program?	|   I stored user input into 3 separtate variables num1, num2 and operation.	|
|  How did you determine what operation to perform?	|  Depending on the operation entered by the user I used a case when syntax to look at the user input and determine which operation to use in the equation. |
|  Do you feel like you used consistent indentation throughout your code? 	|  yes  |
|  If you had more time, what would you have added to or changed about the program? 	|  If I had more time I would have made sure program could pass the tests of dividing my zero and loop back into the equation if the operation was invalid. |

### CODE REVIEW

File: calculator.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Operation
+# two numbers
+# validate the data type from user
+
+puts "\nEnter the first number."
+num1 = gets.chomp
+until num1 =~ /\d/
+    print 'Invalid number. Enter a valid number: '
+    num1 = gets.chomp
+  end
```
Comment (droberts-sea) [inline calculator.rb:36]: Watch your indentation here.

File: calculator.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+puts "\nChoose an Operation"
+operation = gets.chomp
+
+puts "\nEnter the second number."
+num2 = gets.chomp
+until num2 =~ /\d/
+    print 'Invalid number. Enter a valid number: '
```
Comment (droberts-sea) [inline calculator.rb:44]: You've written almost exactly the same code twice here - the only difference is the variable name, `num1` or `num2`. Could you consolidate this somehow?

**Comment by droberts-sea** [comment]
# Calculator
## What We're Looking For

Feature | Feedback
---	| ---
Takes in two numbers and an operator and performs the mathematical operation. | yes
Readable code with consistent indentation. | yes

Great job overall! I have a couple of inline comments below that you should check out, but in general I am quite happy with this submission. Keep up the hard work!
